src/context.rs

```diff
@@ -63,6 +63,7 @@ pub struct Context {
     pub running_state: Arc<RwLock<RunningState>>,
     /// Mutex to avoid generating the key for the user more than once.
     pub generating_key_mutex: Mutex<()>,
+    pub translated_stockstrings: Arc<RwLock<HashMap<usize, String>>>,
```

Comment:
Why does this need to be `Arc<RwLock<>>`?  On first glance at least the `Arc` might not be needed.

For the RwLock, I need to think more about which guarantees rust gives on access, which of those the ffi breaks etc.

Reply:
i just did what is done also with all other attributes in Context :) but indeed seems one can take away the Arc.  Curious why it's used in all other Context attributes?